### PR TITLE
Removes the lower character limit when naming GPS

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
It doesn't make any sense for it to be 4-5 from what I know, so now it's a maximum of 5 characters long so now you can have GPS tags of 1-5 characters instead of 4-5

:cl:
 * tweak: GPS tag lower limit has been removed, so now you can have a tag 1-5 letters long instead of 4-5.

### CODE REVIEW

**Comment by FudgePucker** [comment]
Slip a security officer, force feed them sleep tox, strip their PDA, rename to dab, ????, have your lifeless corpse be tossed into space.
Honk.

File: code/modules/telesci/gps.dm [hunk 1]
```diff
@@ -123,8 +123,8 @@ var/list/SPS_list = list()
 			to_chat(usr, "<span class = 'caution'>The GPS needs to be kept in your active hand!</span>")
 			return TRUE
 		a = strict_ascii(a)
-		if(length(a) < 4 || length(a) > 5)
-			to_chat(usr, "<span class = 'caution'>The tag must be between four and five characters long!</span>")
+		if(length(a) > 5)
```
Comment (jknpj) [inline code/modules/telesci/gps.dm:126]: are you sure you didn't forget to set the min to 1?

Comment (Zth--) [reply code/modules/telesci/gps.dm:126]: what a beautiful var name `a` is. Pretty sure an empty input is being handled by ```if(!a) return TRUE``` a few lines above

Comment (Zth--) [reply code/modules/telesci/gps.dm:126]: hmm wait, if `strict_ascii(a)` removes something then you can end up with an empty variable

**Comment by salamandooo** [comment]
I don't like this. GPS tags should be 4 letters long exactly, anything different is a disgrace

File: code/modules/telesci/gps.dm (hunk 1 above)
Comment (boy2mantwicethefam) [reply code/modules/telesci/gps.dm:126]: ill look into it

**Comment by tuypo1** [comment]
It looks better and is more readable on the gps list the way it is

**Comment by boy2mantwicethefam** [comment]
I've checked it myself, the distance is static

**Pull request merged**